splito/lohi/_lo.py

```diff
@@ -0,0 +1,160 @@
+from rdkit import Chem, DataStructs
+from rdkit.Chem import AllChem
+import numpy as np
+from tqdm import tqdm
+
+from ._utils import get_similar_mols
+
+class LoSplitter():
+    def __init__(self, threshold=0.4, min_cluster_size=5, max_clusters=50, std_threshold=0.60):
+        """
+        Parameters:
+            threshold -- molecules with a similarity higher than this value are considered similar.
+            min_cluster_size -- number of molecules per cluster.
+            max_clusters -- maximum number of selected clusters. The remaining molecules go to the training set.
+            std_threshold -- Lower bound of the acceptable standard deviation for a cluster. It should be greater than measurement noise.
+                            If you're using ChEMBL-like data, set it to 0.60 for logKi and 0.70 for logIC50.
+                            Set it lower if you have a high-quality dataset.
```

Comment:
Thanks for the extensive documentation! That's great! :book: 

This does use a different docstring style. See [this example](https://github.com/datamol-io/splito/blob/main/splito/_mood_split.py#L238-L255). We use `mkdocs` to build and serve the documentation. Could you verify that everything works by running: 

```
mkdocs serve
```

Reply:
Done. Also, I'll probably change the docstrings again when I add a tutorial.